release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -557,6 +557,13 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
 
+[id="ocp-4-17-expose-mtu"]
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings), any heading that's not the top-level heading of an assembly needs a context at the end of its ID:
```suggestion
[id="ocp-4-17-expose-mtu_{context}"]
```

Reply:
Thanks @skopacz1 for your feedback. I have applied your suggestion.